cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/PredicateFunctionsTest.scala

```diff
@@ -81,7 +83,8 @@ class PredicateFunctionsTest extends DocumentingTest {
       }
     }
     section("exists()", "functions-exists") {
-      p("`exists()` returns true if a match for the given pattern exists in the graph, or if the specified property exists in the node, relationship or map.")
+      p("`exists()` returns true if a match for the given pattern exists in the graph, or if the specified property exists in the node, relationship or map." +
+        " Returns `null` if the input argument is `null`.")
```

Comment:
"Returns `null` if the..." -> "`null` is returned if the ..."  (to make it a sentence, rather than a clause)